### DISCUSSION TITLE
How to save segmentation data

### DISCUSSION THREAD

**Discussion by keerti-ds** [microsoft/AirSim#4103] [Support Q&A] [answered]
I am able to run the Unreal engine and AirSim with neighborhood environment. I can also view three different views in sub-windows as shown in the demo video. 

Now, I want to save the segmented view for further usage(either in video or image format). I could not find out much on this. 
Any suggestions or hints on the same? How can I achieve this.

Thanks in advance.

**Comment by keerti-ds** [comment]
I am able to save them successfully, I had to modify `segmentation.py `

However, I was wondering if its possible to save these images dynamically. Now I can only save a single image when I run `segmentation.py`. Something like in loop.

**Comment by kulkarnikeerti** [comment, marked as answer]
It could be achieved using `cv_capture.py`
